data-prepper-plugins/otel-metrics-raw-processor/src/main/java/com/amazon/dataprepper/plugins/processor/otelmetrics/model/RawSummary.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.amazon.dataprepper.plugins.processor.otelmetrics.model;
+
+import com.fasterxml.jackson.annotation.JsonAnyGetter;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.opentelemetry.proto.metrics.v1.SummaryDataPoint;
+
+import java.util.List;
+import java.util.Map;
+
+public class RawSummary {
+
+    private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
+
+    private final Map<String, Object> attributes;
+    private final String kind;
+    private final String unit;
+    private final String name;
+    private final String description;
+    private final String startTime;
+    private final String time;
+    private final String serviceName;
+    private final int quantileValuesCount;
+    private final List<SummaryDataPoint.ValueAtQuantile> quantileValuesList;
```

Comment:
Perhaps we should rename this to `quantileValues`. The Trace models don't have a `List` suffix on property names.

Reply:
Done